### PR TITLE
New version of readme

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,32 @@
-# Area FiftyLAN Website
-Welcome to the pretty part of the Area FiftyLAN system! This repo contains all the code that you will see served at https://areafiftylan.nl, including the ticketsale, my-area and soon the seatmap.
+# LANcie frontend
+Welcome to the pretty part of the Area FiftyLAN system! This repo contains all the code that you will see served at https://areafiftylan.nl, including the ticketsale, my-area and seatmap. The front-end is built with Polymer, Google's Web Components framework. There are a couple of things you need to do before you can enjoy all that lovely material design goodness.
```
Comment (MatthijsKok) [inline README.md]: Looks really nice over all. Maybe add link to polymer homepage back?

File: README.md [hunk 2]
```diff
@@ -1,47 +1,32 @@
...
+### Build
+To build the entire frontend, the `npm run build` command is used. This places the complete and ready to serve webpage in `./build`. There are two folders located in the build folder, a bundled and an unbundled version. The bundled version is for non-http/2 compatible servers. The unbundled version is for http/2 compatible servers. Keep in mind that if images are added since the last build, these have to be compressed, this is done with `npm run build optimize-images`
 
-Easy enough huh?
+### Deploy
+To deploy the frontend, you can upload it to a server. This can be done by copying and pasting manually, but this is quite cumbersome. The easiest way to do this is with the `scp` command (only available on linux and OS X, no download needed). The scp command works like this:
 
-There's one more catch. While the homepage can do fine without someone to talk to, the more complex functionality like buying tickets and My Area require a back-end. See [the LANcie API](https://github.com/AreaFiftyLAN/lancie-api) for details on how to run that. We expect it to run at port 9000!
+```scp -r $(local_build_folder)/. $(server_login)@$(server_ip):$(server_destination_folder)```
```
Comment (TimvdLippe) [inline README.md]: Shouldnt it be /* for the local build folder?

Comment (martijnjanssen) [inline README.md]: It does the same, `*` is a wildcard and matches all the folder contents. The `.` indicates the content of the folder.

Comment (TimvdLippe) [inline README.md]: All right, thanks for the explanation!

**Pull request merged**